lib/registries/addon/drafts/draft/-components/right-nav/template.hbs

```diff
@@ -7,12 +7,13 @@
         data-test-goto-review
         data-analytics-name='Go to review'
         local-class='ReviewButton'
-        class='btn btn-primary'
         @type='button'
         @route='registries.drafts.draft.review'
         @models={{array @draftManager.draftId}}
     >
-        {{t 'registries.drafts.draft.review.start_review'}}
+        <Button @type='primary' @layout='large'>
```

Comment:
oh i see, this is the inverse of the `fake-link` button-that-looks-like-a-link -- it's a link that looks like a button...

i see this as a bit of debt we'll have to figure out (ideally we'd separate the routing logic from how the element is visually displayed, somehow...) but this works with keyboard navigation and seems fine for now